hack/fix-helm-chart-version.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
would rather name the file `update-helm-chart-version` or `set-helm-chart-version`.
Fix does more indicate some error

Reply:
`update-helm-chart-version.sh` it is.